proc_scripts/plotting.py

```diff
@@ -95,10 +95,11 @@ def side_by_side(self,plotname,s,thisrange):
         self.next(plotname, fig=thisfig)
         sca(ax1)
         forplot = s['t2':expand_limits(thisrange,s)]
-        self.image(forplot)
+        self.image(forplot.C.setaxis('power','#').set_units('power','scan #'))
         draw_limits(thisrange,forplot)
         sca(ax2)
-        self.image(forplot.C.cropped_log())
+        self.image(forplot.C.cropped_log().C.setaxis(
+'power','#').set_units('power','scan #'))
```

Comment:
indentation here is messed up. highlight and press = to auto-fix it.

Reply:
done